packages/run-protocol/src/runStake/runStake.js

```diff
@@ -0,0 +1,125 @@
+// @ts-check
+import { handleParamGovernance } from '@agoric/governance';
+import { E, Far } from '@endo/far';
+import { makeAttestationFacets } from './attestation.js';
+import { makeRunStakeParamManager } from './params.js';
+import { makeRUNstakeKit } from './runStakeKit.js';
+import { makeRunStakeManager } from './runStakeManager';
+
+/**
+ * @param { ZCF<RunStakeTerms> } zcf
+ * @param {{
+ *   feeMintAccess: FeeMintAccess,
+ *   initialPoserInvitation: Invitation,
+ *   lienBridge: ERef<StakingAuthority>,
+ * }} privateArgs
+ *
+ * @typedef {{
+ *   timerService: TimerService,
+ *   chargingPeriod: bigint,
+ *   recordingPeriod: bigint,
+ *   lienAttestationName: string,
+ *   electionManager: VoteOnParamChange,
+ *   main: {
+ *     MintingRatio: ParamRecord<'ratio'>,
+ *     InterestRate: ParamRecord<'ratio'>,
+ *     LoanFee: ParamRecord<'ratio'>,
+ *     Electorate: ParamRecord<'invitation'>,
+ *   },
+ * }} RunStakeTerms
+ */
+const start = async (
+  zcf,
+  { feeMintAccess, initialPoserInvitation, lienBridge },
+) => {
+  const {
+    main: initialValue,
+    brands: { Stake: stakeBrand },
+    timerService,
+    chargingPeriod,
+    recordingPeriod,
+    lienAttestationName = 'BldLienAtt',
+  } = zcf.getTerms();
+  assert.typeof(chargingPeriod, 'bigint', 'chargingPeriod must be a bigint');
+  assert.typeof(recordingPeriod, 'bigint', 'recordingPeriod must be a bigint');
+
+  const att = await makeAttestationFacets(
+    zcf,
+    stakeBrand,
+    lienAttestationName,
+    lienBridge,
+  );
+  // TODO: remove publicFacet wart
+  const attestBrand = await E(att.publicFacet).getBrand();
+
+  const paramManager = await makeRunStakeParamManager(
```

Comment:
https://github.com/Agoric/agoric-sdk/pull/4868 will make it so you don't have to hold the full `paramManager` or check that electorate matches.